bool master_lock  # true = ตัดการควบคุมจากคนถาวร, false = ปกติ
---
bool success
string message